### PR TITLE
Add Epson Projector as new OS

### PR DESCRIPTION
**Pull request by mpikzink**
Inspired by
https://community.librenms.org/t/new-device-epson-network-projector/11631

But I haven't found MIBS...

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/projector.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    cisco-otv: false
+    cisco-pw: false
+    cisco-sla: false
+    cisco-vrf-lite: false
+    vrf: false
+    libvirt-vminfo: false
+    ntp: false
+    stp: false
+    ucd-diskio: false
+    vlans: false
```
Comment (Jellyfrog) [inline includes/definitions/projector.yaml:46]: Please remove all these if not absolutely needed

File: includes/discovery/sensors/count/projector.inc.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * @package    LibreNMS
+ * @subpackage webui
+ * @link       http://librenms.org
+ * @copyright  2018 LibreNMS
+ * @author     LibreNMS Contributors
+*/
+
+use Illuminate\Support\Str;
+
+if (Str::startsWith($device['sysObjectID'], '.1.3.6.1.4.1.1248.4.1')) {
```
Comment (Jellyfrog) [inline includes/discovery/sensors/count/projector.inc.php:20]: This should be possible to implement in yaml, would you mind giving it a try?

Comment (mpikzink) [reply includes/discovery/sensors/count/projector.inc.php:20]: Is this possible without mibs? Haven't found some so I did this workaround.

Comment (Jellyfrog) [reply includes/discovery/sensors/count/projector.inc.php:20]: Hmm I think you can just use the oids

Comment (mpikzink) [reply includes/discovery/sensors/count/projector.inc.php:20]: OK, sure then I will try it of course

Comment (mpikzink) [reply includes/discovery/sensors/count/projector.inc.php:20]: `
modules:
    sensors:
        count:
            data:
                -
                    num_oid: '.1.3.6.1.4.1.1248.4.1.1.1.1.{{ $index }}'
                    descr: Lamp Hours
		    index: 0
`
Does nor work :-(

Comment (Jellyfrog) [reply includes/discovery/sensors/count/projector.inc.php:20]: Need to add Oid also I guess

Comment (mpikzink) [reply includes/discovery/sensors/count/projector.inc.php:20]: unfortunately also does not work with Numeric OIDs

Comment (murrant) [reply includes/discovery/sensors/count/projector.inc.php:20]: the sensors module does not support numeric only in yaml :/

File: includes/definitions/projector.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+os: projector
```
Comment (Jellyfrog) [inline includes/definitions/projector.yaml:1]: I don't think we should call the OS `projector`, since it's unique for Epson

Comment (mpikzink) [reply includes/definitions/projector.yaml:1]: good point I will change to epson_projector

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**